splash/lua_modules/wraputils.lua

```diff
@@ -134,32 +134,33 @@ end
 -- Handle @lua_property decorators.
 --
 local function setup_property_access(py_object, self, cls)
-  local setters = {}
-  local getters = {}
+  self.setters = {}
+  self.getters = {}
```

Comment:
I suppose that we should somehow hide this objects from `self`. The previous closure method didn't work.